wgengine/magicsock/magicsock_test.go

```diff
@@ -464,6 +458,12 @@ func TestTwoDevicePing(t *testing.T) {
 		ping2(t)
 	})
 
+	if os.Getenv("RUN_CURSED_TESTS") == "" {
+		t.Skip("test is very broken, don't run in CI until it's reliable.")
+	}
+	defer dev1.Close() // TODO(crawshaw): this hangs
```

Comment:
This doesn't hang any more, that TODO is outdated. You can move those back into the main portion of the test.